Sources/TestSupport/misc.swift

```diff
@@ -72,11 +72,12 @@ public func fixture(name: String, tags: [String] = [], file: StaticString = #fil
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "init"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.email", "example@example.com"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.name", "Example Example"])
+                try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "commit.gpgsign", "false"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "add", "."])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "commit", "-m", "msg"])
```

Comment:
Yeah I recently added commonly used git commands to an extension in `TestSupport` (That module is only available to tests) https://github.com/apple/swift-package-manager/pull/711
so this can be simplified to:
```swift
try systemQuietly([Git.tool, "-C", dstDir.asString, "init"])
let repo = GitRepository(path: dstDir)
repo.stageEverything()
repo.commit()
```